po/extra/pt_BR.po

```diff
@@ -8,399 +8,440 @@ msgstr ""
 "Project-Id-Version: extra\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-02-08 18:00-0500\n"
-"PO-Revision-Date: 2020-02-03 06:04-0500\n"
-"Last-Translator: Automatically generated\n"
+"PO-Revision-Date: 2020-02-08 20:30-0300\n"
+"Last-Translator: Gleisson J.J. Cardoso <gleissoncg2@gmail.com>\n"
 "Language-Team: none\n"
 "Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.0.6\n"
 
 #: data/com.github.alainm23.planner.desktop.in:3
 #: data/com.github.alainm23.planner.desktop.in:4
 #: data/com.github.alainm23.planner.appdata.xml.in:7
 msgid "Planner"
-msgstr ""
+msgstr "Planejador"
 
 #: data/com.github.alainm23.planner.desktop.in:5
 msgid "Never worry about forgetting things again"
-msgstr ""
+msgstr "Não se preocupe em esquecer algo novamente"
 
 #: data/com.github.alainm23.planner.desktop.in:8
 msgid "com.github.alainm23.planner"
-msgstr ""
+msgstr "com.github.alainm23.planner"
 
 #: data/com.github.alainm23.planner.desktop.in:12
 msgid "development;task;tasks;project;todo;event;events;calendar;todoist;"
 msgstr ""
+"desenvolvimento; tarefa; tarefas; projeto; todo; evento; eventos; "
```

Comment:
Do not separate the elements with space.

Reply:
Solved.